src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -15,7 +15,8 @@ const CPM_ABILITIES = [
     spell: SPELLS.PRAYER_OF_MENDING_CAST,
     name: 'Prayer of Mending',
     category: SPELL_CATEGORY.ROTATIONAL,
-    getCooldown: (haste, combatant) => (12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) / (1 + haste),
+    getCooldown: (haste, combatant) => ((12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) + 1.5) / (1 + haste), // +1.5 for base cast time
+    recommendedCastEfficiency: 0.70, // There's a lot of things that can potentially delay PoM: movement, heavy damage, etc. It is important to maintain high uptime but delaying a few GCDs is not bad.
```

Comment:
I am looking at live Sir, I figured it may have been that it was not pushed out yet. Great to know it will be adjusted soon then. I was talking to my guild about it and wanted to make sure I made someone aware in case. :)